setup.py

```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="dataEng-container-tools",
-    version="0.6.1",
+    version="0.6.4",
     author="Alexander Saff",
     author_email="alexander_saff@colpal.com",
```

Comment:
should we update this? Alex has not worked at Colgate for some time. what do you think @mourtallah  ?

Reply:
just from a support perspective, might make sense to add DE google group or something instead of Alex's email

Reply:
I was thinking the same thing for the PKG-INFO as well, maybe also changing the name to the DE team.

Reply:
i can see that. whats the email?

Reply:
I've updated the contact information